config/__init__.py

```diff
@@ -119,6 +119,32 @@ def update_status_config(milestones_content):
     return STATUSES
 
 
+def get_variation_from_env(envvar_value):
+    """Return a tuple with variation data from the specific environment
+    variable key.
+
+    Raise an exception if the passed in envvar doesn't look something
+    like: '0 100'
+    """Convert a string to a tuple of integers.
```

Comment:
Oops, looks like GitHub's accept suggestion thingy got confused. 💣

Reply:
yeah the suggestion works for one line only.